Makefile

```
@@ -0,0 +1,45 @@
+# Copyright (c) 2025 Hemi Labs, Inc.
+# Use of this source code is governed by the MIT License,
+# which can be found in the LICENSE file.
+
+.PHONY: all clean deps tidy build lint lint-deps test
+
+all: tidy build lint test
+
+clean:
+	rm -rf ./bin/
+
+deps: tidy
+	go mod download
+	go mod verify
+
+tidy:
+	go mod tidy
+
+build:
+	go build -trimpath -ldflags "-s -w $(GO_LDFLAGS)" ./larry
+
+define LICENSE_HEADER
+Copyright (c) 2025 Hemi Labs, Inc.
```

golicenser will replace a `{{.year}}` template variable with the year (git-range will create a range using information from Git)
```suggestion
Copyright (c) {{.year}} Hemi Labs, Inc.
```